### PR TITLE
docs: webwallet docs for all supported features

### LINKED ISSUE #406: web wallet : blinded routing
- [x]  setup messenger service for sending messages and replies #407 
- [x]  setup correlation between message sent and reply received #405 
- [x]  add share router peer DID step during wallet did connect #408
- [x]  request peer DID from other party (RP or Issuer) #409
- [x]  request router peer DID from router #410
- [x]  share router peer DID to other party (RP or Issuer) #411
- [x] integration #434
- [x] bdd tests #425 
- [x] documentation #461

### LINKED ISSUE #461: web wallet: add more documentation for supported features

### PR DESCRIPTION
**Pull request by sudeshrshetty**
- Closes #406
- Closes #461

Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

File: docs/components/web_wallet.md [hunk 1]
```diff
@@ -34,3 +34,363 @@ User agent web wallet supports various wallet operations and also [DIDComm](http
 - **DID Connect**: This operation can be used by a website(or agent) to perform [DID Exchange](https://github.com/hyperledger/aries-rfcs/blob/master/features/0023-did-exchange/README.md) with web wallet for all future DID Comm operations.  
 - **Presentation Exchange**: This operation can be used by a website (a Relying Party or verifier) to request submission of proof from wallet in align with requester's [proof requirements](https://identity.foundation/presentation-exchange/). This request can also be made in conjuction with DID Connect operation, so that wallet can request further information from requester (a Relying party or verifier) over DID comm channel.
 
+## Login to wallet
+After successful login, web wallet will load [polyfill library](https://github.com/digitalbazaar/credential-handler-polyfill) and a web credential handler will be installed which will serve all your credential handler request as per [Credential Handler API](https://w3c-ccg.github.io/credential-handler-api/) (aka CHAPI) standards.
+
+In addition to that, user agent web wallet will perform some onboarding operations during first login of an user to support various DID comm operations which includes,
```
Comment (llorllale) [inline docs/components/web_wallet.md:40]: These onboarding operations will change once the TrustBloc login & recovery components are up and running.

cc @Baha-sk @aholovko @DRK3

File: docs/components/web_wallet.md [hunk 2]
```diff
@@ -34,3 +34,363 @@ User agent web wallet supports various wallet operations and also [DIDComm](http
...
+ });
+
+const result = await navigator.credentials.store(webCredential);
+if(!result) {
+  console.log('store credential operation did not succeed');
+}
+
+```
+
+Note: while storing, credential(s) always has to be in presentation format
```
Comment (llorllale) [inline docs/components/web_wallet.md:66]: what are the supported formats?

File: docs/components/web_wallet.md [hunk 3]
```diff
@@ -34,3 +34,363 @@ User agent web wallet supports various wallet operations and also [DIDComm](http
...
+## Getting credential from wallet
+Credentials can be requested from web wallet through `get()` call using a CHAPI request as given in sample below,
+
+```javascript
+let credentialQuery = {
+  "web": {
+    "VerifiablePresentation": {
+      "query": [
+        {
+          "type": "QueryByExample",
```
Comment (llorllale) [inline docs/components/web_wallet.md:77]: This type is not supported by the wallet as far as I can see.

We should put more representative examples and point to their documentation.

**Review by llorllale — APPROVED** [review]

File: docs/components/web_wallet.md [hunk 4]
```diff
@@ -34,3 +34,363 @@ User agent web wallet supports various wallet operations and also [DIDComm](http
...
+                                      "domain": "example.com"
+                                    }
+                                  }
+                                }
+
+const webCredential = await navigator.credentials.get(presentationExchangeQuery);
+
+```
+
+## DID Comm by web wallet
```
Comment (rolsonquadras) [inline docs/components/web_wallet.md:261]: DID Comm -> DIDComm (single word).

File: Makefile [hunk 5]
```diff
@@ -72,6 +72,11 @@ generate-test-keys:
 user-agent-start: clean user-agent-wasm-docker generate-test-config generate-test-keys mock-login-consent-docker
 	@scripts/user_agent_start.sh
 
+# starting user agent in dev mode for hot deployment
+.PHONY: user-agent-dev-start
+user-agent-dev-start:
+	@scripts/user_agent_dev_start.sh
```
Comment (rolsonquadras) [inline Makefile:78]: Missed to check-in the file ? I don't see this file in this PR nor in main branch.

File: docs/components/web_wallet.md (hunk 1 above)
Comment (sudeshrshetty) [reply docs/components/web_wallet.md:40]: Yes

File: Makefile (hunk 5 above)
Comment (sudeshrshetty) [reply Makefile:78]: Oh missed it, will add it

File: docs/components/web_wallet.md (hunk 4 above)
Comment (sudeshrshetty) [reply docs/components/web_wallet.md:261]: Fixed

File: Makefile (hunk 5 above)
Comment (sudeshrshetty) [reply Makefile:78]: fixed

**Review by rolsonquadras — APPROVED** [review]

File: docs/components/web_wallet.md (hunk 2 above)
Comment (sudeshrshetty) [reply docs/components/web_wallet.md]: updated the docs

**Pull request merged**